ansible/inventories/monolith

```diff
@@ -25,6 +25,12 @@
 [rabbitmq]
 192.168.33.21
 
+[zookeeper]
+192.168.33.21
+
+[kafka]
+192.168.33.21 kafka_broker_id=0
```

Comment:
modeled after how elasticsearch cluster names worked (each broker_id must be unique to the cluster)